app/jobs/gpo_expiration_job.rb

```diff
@@ -39,6 +44,14 @@ def analytics
     @analytics ||= Analytics.new(user: AnonymousUser.new, request: nil, session: {}, sp: nil)
   end
 
+  def are_old_enough(as_of:, min_profile_age:)
+    return Profile.where('1=1') if min_profile_age.blank?
```

Comment:
Curious why `.where('1=1')` instead of `.all`?

Reply:
Because I forgot about `.all` 🤣 ...just a sec, will fix

Reply:
Fixed in f0d3e77fe2f0f2a567ee50a33ba0f89e04163a2e